src/app/static/src/js/components/runReport/runReportTabs.vue

```diff
@@ -38,9 +38,7 @@
             <div class="col-12 col-md-8 tab-content">
                 <div v-if="selectedTab === 'runReport'" class="tab-pane active pt-4 pt-md-1" role="tabpanel" id="run-tab">
                     <div id="runReportVueApp">
-                        <run-report :metadata="metadata"
-                                    :initial-git-branches="initialGitBranches"
-                                    :initial-report-name="initialReportName"
+                        <run-report :initial-report-name="initialReportName"
```

Comment:
I've left this as a prop for now just because it isn't returned from the endpoint in the way the metadata is. We could change it to read from the url instead if we want, or leave it as a prop (I think it's only used by one component, so maybe doesn't need to be in the Vuex store and can remain as a simple prop like this)